index.html

```diff
@@ -1,15 +1,341 @@
 <!DOCTYPE html>
 <html lang="fr">
-  <head>
+
+<head>
     <meta charset="utf-8" />
     <title>Booki</title>
```

Comment:
put after viewport

Reply:
I put css link under meta tags, please review and let me think if it's good. Thanks